zenoh-kotlin/build.gradle.kts

```diff
@@ -106,9 +157,32 @@ kotlin {
                     password = System.getenv("GITHUB_TOKEN")
                 }
             }
+            maven {
+                name = "MavenCentral"
+                url = uri(if (project.hasProperty("SNAPSHOT"))
+                    "https://oss.sonatype.org/content/repositories/snapshots/"
+                else
+                    "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
+                )
```

Comment:
Shouldn't this be reversed? If it's a snapshot, publish to the staging repository?

Reply:
I think it's ok as it is, based on the documentation from https://oss.sonatype.org/content/repositories/snapshots/